### ISSUE TITLE
Global ProjectDeveloper role is not needed

### ISSUE THREAD

**Issue by lucyb** [opensafely-core/job-server#4520]

**Comment by Jongmassey** [comment]
https://github.com/opensafely-core/job-server/pull/4540

**Comment by Jongmassey** [comment]
Based on today's tech team meeting - it appears that there is still a need for tech-support to run jobs within projects and this is permissible under the applicable policies (per @CLStables ). 

I therefore propose we create a new global tech support role with only the required permissions and use that in place of the global `ProjectDeveloper` role (whilst also removing the ability to associate the `ProjectDeveloper` role with a user).

**Comment by lucyb** [comment]
> I therefore propose we create a new global tech support role with only the required permissions and use that in place of the global ProjectDeveloper role

I agree. I would like to suggest that the new tech support role only has the `job_run` permission to start with (note: jobs can be cancelled through the staff area). It's easier to add permissions to a role than to take them away, so if we discover that people on tech support need to do more, then we can always give additional permissions (in line with the developer permissions logs).

We should do that as part of this ticket. We should also ensure the tech support section in the team manual says that tech support members require this role and can re-create job requests in the case of a technical problem.

**Comment by Jongmassey** [comment]
> It's easier to add permissions to a role than to take them away, so if we discover that people on tech support need to do more, then we can always give additional permissions (in line with the developer permissions logs).

I agree

**Comment by Jongmassey** [comment]
The only mention I can find of the global `ProjectDeveloper` role in the team manual is here:

>Technical Notes[¶](https://bennett.wiki/products/data-access-policy-implementation/#technical-notes_3)
Access to run jobs and view outputs on any project in Job Server is granted by assigning the global ProjectDeveloper and ProjectCollaborator roles to that user
For the avoidance of doubt, a ProjectDeveloper role does not also need to be added at the project level unless the user is also acting as a researcher on that project (see above)

We should discuss with @CLStables as to how best to align the Tech Support role with the developer permissions in the developer permissions log and how best to update this above guidance.

**Comment by Jongmassey** [comment]
Simplest thing for now was to just replace "global `ProjectDeveloper`" with "global `TechSupport`" in the docs for now. 

Global `ProjectCollaborator` should also be given consideration as to its appropriateness in light of this change and the recent `OutputChecker` changes.

https://github.com/ebmdatalab/team-manual/pull/832
